modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/info/item/provider/test/ObjectEntryInfoItemFormProviderTest.java

```diff
@@ -465,6 +339,239 @@ private ServiceContext _getServiceContext(ObjectEntry objectEntry)
 		return serviceContext;
 	}
 
+	private void _testGetInfoFormWithAttachmentObjectField() throws Exception {
+		InfoItemFormProvider<?> infoItemFormProvider =
```

Comment:
can we put this in the setUp method? Or put it in the main method (`testGetInfoForm`) and pass as a param

Maybe having `childInfoForm` and `parentInfoForm` in the main method? Isolate it somehow.

```
nfoItemFormProvider<?> infoItemFormProvider =
			_infoItemServiceRegistry.getFirstInfoItemService(
				InfoItemFormProvider.class,
				_childObjectDefinition.getClassName());

		InfoForm infoForm = infoItemFormProvider.getInfoForm(
			String.valueOf(_childObjectDefinition.getObjectDefinitionId()), 0);
```